sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -978,11 +1070,7 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     testDropTable(isDatasourceTable = false)
   }
 
-  test("drop table - data source table") {
-    testDropTable(isDatasourceTable = true)
-  }
-
-  private def testDropTable(isDatasourceTable: Boolean): Unit = {
+  protected def testDropTable(isDatasourceTable: Boolean): Unit = {
```

Comment:
This is not only for `InMemoryCatalogedDDLSuite `. We have another call [here](https://github.com/gatorsmile/spark/blob/ec914dcad3ac4e360e4a9b5395a226f94ae5715b/sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala#L1070).